src/components/Navbar.js

```diff
@@ -0,0 +1,255 @@
+import { Link } from 'react-router-dom';
+import { Fragment } from 'react';
+import { Disclosure, Menu, Transition } from '@headlessui/react';
+import { MenuIcon, XIcon } from '@heroicons/react/outline';
+import logo from '../assets/logov.png';
+

```

Comment:
Let's have an empty line between local import and others. 
```suggestion
import { Link } from 'react-router-dom';
import { Fragment } from 'react';
import { Disclosure, Menu, Transition } from '@headlessui/react';
import { MenuIcon, XIcon } from '@heroicons/react/outline';

import logo from '../assets/logov.png';
```